views/help/partials/help_first_steps.hbs

```diff
@@ -2,12 +2,12 @@
 	{{#content "card-body"}}
 		{{> "help/icon-link-group" link-class="col-sm-3" links=(jsonParse '[
 			{
-				"title": "Schüler",
+				"title": "{{$t "help.global.heading.student" }}",
```

Comment:
Same thing here:
You can't call a helper inside of a string...
I would suggest handling this with the helper (concat 'string' ($t "some.key") 'someMorestring')
That would not really be fancy, but it would do the trick.

Reply:
done ;)